hypex/dataset/backends/pandas_backend.py

```diff
@@ -190,11 +191,27 @@ def _get_column_index(
             else self.data.columns.get_indexer(column_name)
         )[0]
 
-    def get_column_type(self, column_name: str) -> str:
-        return str(self.data.dtypes[column_name])
+    def get_column_type(self, column_name: str) -> Union[DefaultRoleTypes, None]:
```

Comment:
This method does not return objects of DefaultRoleTypes, i.e. int, float or bool, it returnes type name